fc/fc_app.py

```diff
@@ -9,13 +9,12 @@
 4. datacube-fc generate
```

Comment:
This needs to be updated given that it seems you've removed the `ensure-products` and `generate` cli commands (and also anywhere else that my mention/use them)